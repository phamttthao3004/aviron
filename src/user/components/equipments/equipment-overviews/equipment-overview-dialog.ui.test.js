import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import EquipmentOverviewDialogUI from './equipment-overview-dialog.ui';

describe('Equipment Overview Dialog UI', () => {
    it('Render Equipment Overview Dialog UI component', () => {
        const props = { state: undefined, setOpenEdit: undefined };
        const com = shallow(<EquipmentOverviewDialogUI {...props} />);
        expect(shallowToJson(com)).toMatchSnapshot();
    });
});
