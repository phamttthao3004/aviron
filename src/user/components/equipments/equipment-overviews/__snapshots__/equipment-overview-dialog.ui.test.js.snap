// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Equipment Overview Dialog UI Render Equipment Overview Dialog UI component 1`] = `
<CmsDialog
  className="dialog-add-edit-group"
  open={false}
>
  <div
    className="cms-dialog-content"
  >
    <div
      className="cms-dialog-title"
    >
      <span>
        EDIT
      </span>
    </div>
    <div
      className="cms-dialog-main"
    >
      <div
        className="modal-body edit-group-container"
      >
        <p>
          EQUIPMENT NAME
        </p>
        <input
          className="form-control"
          placeholder="Equipment name"
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      className="cms-dialog-footer"
    >
      <button
        className="btn btn-default"
        name="button"
        onClick={[Function]}
        type="button"
      >
        Cancel
      </button>
      <button
        className="btn btn-primary"
        name="button"
        type="button"
      >
        Add
      </button>
    </div>
  </div>
</CmsDialog>
`;
